SYCL/SpecConstants/2020/non_native/SpecConstBuffer.cpp

```diff
@@ -0,0 +1,25 @@
+// REQUIRES: opencl-aot, cpu
+
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_x86_64 %S/Inputs/common.cpp -o %t.out \
+// RUN:          -fsycl-dead-args-optimization
+// RUN: env SYCL_PI_TRACE=-1 %CPU_RUN_PLACEHOLDER %t.out | FileCheck %s
+
+#include <cmath>
+#include <sycl/sycl.hpp>
+
+class Kernel1Name;
```

Comment:
This is redundant as you use it as `class Kernel1Name` in `single_task`.

Reply:
Yes, I've already removed it.